docker-compose.yml

```diff
@@ -40,7 +39,6 @@ services:
       - db
     links:
       - db
-    restart: always
```

Comment:
when the server is restarted for any reason this helps start the service along with the database. hence we shouldn't remove it.